src/deepqmc/wf/paulinet/paulinet.py

```diff
@@ -46,79 +44,51 @@ def eval_log_slater(xs):
 
 class PauliNet(WaveFunction):
     r"""
-    Implement the PauliNet Ansatz from [HermannNC20]_.
+    Implements the neural network wave function.
 
     Args:
-        hamil (~jax.MolecularHamiltonian): the Hamiltonian of the system.
-        basis (~jax.wf.paulinet.env.ExponentialEnvelopes): optional, the orbital
-            envelopes.
-        confs (int, (:math:`N_\text{conf}`, :math:`N_\text{elec}`)): optional,
-            specifies the electronic configuration, i.e. the occupied orbitals
-            in each determinant. The value of :data:`confs[i,j]` is the index of
-            the orbital occupied by the :data:`j` th electron in the :data:`i` th
-            determinant.
-        backflow_op (Callable): optional, specifies how the backflow is applied
-            to the orbitals.
-        full_determinant (bool): optional, if :data:`False`, the determinants are
+        hamil (~MolecularHamiltonian): the Hamiltonian of the system.
+        basis (~wf.paulinet.env.ExponentialEnvelopes): the orbital envelopes.
```

Comment:
Maybe its time to rename this, e.g. to `envelope`, since this has not much to do anymore with an AO basis.

Reply:
Yes I agree. Basis might be confusing.